assemblers/org.osbuild.oci-archive

```diff
@@ -103,7 +103,7 @@ MEDIA_TYPES = {
 
 def sha256sum(path: str) -> str:
     ret = subprocess.run(["sha256sum", path],
-                         capture_output=True,
+                         stdout=subprocess.PIPE,
                          encoding="utf-8",
                          check=True)
```

Comment:
There is also `subprocess.check_output()` which sets `check=True` and `stdout=subprocess.PIPE` and returns `stdout`. I am fine with the fix, just mentioning this for the future.

Reply:
I wanted a minimal change. Maybe we could cleanup all the uses on a separate PR.